diff --git a/lib/index.cjs b/lib/index.cjs
index 3e58957730be864e8e4e5fd7ecf0b3fa2f94f8e4..cc2bdf3d811a5e3d9aa83761d499f14ce38f5538 100644
--- a/lib/index.cjs
+++ b/lib/index.cjs
@@ -73,7 +73,7 @@ const n = null, {min: r, max: s, abs: i, floor: l} = Math, c = (e, t, o) => r(o,
             let t, o;
             if (b) [t, o] = R; else {
                 let n = s(0, H()), i = n + a;
-                c || (e = s(0, e), 1 !== x && (n -= e), 2 !== x && (i += e)), [t, o] = R = W(s(0, n), s(0, i)), 
+                c || (e = s(0, e), n -= e, i += e), [t, o] = R = W(s(0, n), s(0, i)), 
                 k && (t = r(t, k[0]), o = s(o, k[1]));
             }
             return [ s(t, 0), r(o, M.l - 1) ];
diff --git a/lib/index.js b/lib/index.js
index 76feb0ffe34ebaa48d410ce9c9ea0c93154ce907..3dee1c9a592fdce7603db0e59be3d9f5645d3f1d 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -77,7 +77,7 @@ const d = null, {min: a, max: h, abs: p, floor: g} = Math, v = (e, t, o) => a(o,
             let t, o;
             if (v) [t, o] = k; else {
                 let n = h(0, H()), i = n + l;
-                r || (e = h(0, e), 1 !== _ && (n -= e), 2 !== _ && (i += e)), [t, o] = k = W(h(0, n), h(0, i)), 
+                r || (e = h(0, e), n -= e, i += e), [t, o] = k = W(h(0, n), h(0, i)), 
                 S && (t = a(t, S[0]), o = h(o, S[1]));
             }
             return [ h(t, 0), a(o, C.l - 1) ];
